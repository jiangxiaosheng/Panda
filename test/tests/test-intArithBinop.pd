func main() {
	var i1 = 1
    var i2 = 2

    i1 + i2
    i1 - i2
    i1 * i2
    i1 / i2
}