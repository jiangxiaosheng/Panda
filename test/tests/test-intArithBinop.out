Sementically checked program: 


void main() {
var i1: int(int : 1)
var i2: int(int : 2)
(int : (int : i1) + (int : i2))
(int : (int : i1) - (int : i2))
(int : (int : i1) * (int : i2))
(int : (int : i1) / (int : i2))
}